src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1621,7 +1698,14 @@ private void UpdateAlreadyPlacedVisuals()
                 }
 
                 var hexNode = placedHexes[nextFreeHex++];
-                hexNode.MaterialOverride = organelle.PlacedThisSession ? validMaterial : oldMaterial;
+                if (islands.Contains(organelle.Position))
+                {
+                    hexNode.MaterialOverride = islandMaterial;
+                    hexesToResetToIslandMaterial.Add(hexNode);
+                }
+                else
+                    hexNode.MaterialOverride = organelle.PlacedThisSession ? validMaterial : oldMaterial;
+
```

Comment:
Use braces here as well.
and also make the ? into if clauses
```
}
else if (organelle.PlacedThisSession)
{
hexNode.MaterialOverride = validMaterial;
}
else
{
hexNode.MaterialOverride = oldMaterial;
}
```

Also for bulletproofness I'd like that resetList.Remove(hexNode) stuff to be added to each of the if bodies (unless that becomes too slow).